medical_medicament/models/product_template.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, _
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    is_medicament = fields.Boolean(
```

Comment:
Good call, surprised I missed this when going through and adding all the helps too. This time around we focused on making sure everything had solid string/help attributes, so please let me know if you notice anything else like this. Goal on this release is for a true production-ready version :smile: